content/en/docs/concepts/storage/volumes.md

```diff
@@ -605,6 +605,42 @@ spec:
       type: Directory
 ```
 
+{{< caution >}}
+It should be noted that the `FileOrCreate` mode does not create the parent directory of the file. If the parent directory of the mounted file does not exist, the pod fails to start. To ensure that this mode works, you can try to mount directories and files separately, as shown below
+{{< /caution >}}
+
+#### Example Pod(FileOrCreate)
```

Comment:
Why is `FileOrCreate` in round brackets? Is that style also used elsewhere? It looks a bit odd to me, I would prefer "Example Pod FileOrCreate".

Reply:
Done. My mother tongue is Chinese and English is a bit awkward